packages/cli/src/lib/webdriver.test.ts

```diff
@@ -22,7 +23,9 @@ describe('startDriver', () => {
   });
 
   afterEach(async () => {
-    await driver.quit();
+    try {
+      await driver.quit();
+    } catch (error) {}
```

Comment:
Is the try-catch required here?

Reply:
Ya it looks like this is required since driver isn't built properly due to mocked chrome options

Reply:
So we don't forget can we add comments to why it was needed and for [L114](https://github.com/dequelabs/axe-core-npm/pull/1038/files#diff-48a6ca6c15e8bb1212533365531f7a5e95434f92acf588c23e62fb2c64f7fdaeR114)